db/migrate/20160428151421_create_privileges.rb

```diff
@@ -1,8 +0,0 @@
-class CreatePrivileges < ActiveRecord::Migration
```

Comment:
Never delete migration files - just create a new migration removing the unwanted fields/tables. This one is _probably_ done with, but deleting migration files has the potential to make things go _badly_ wrong.

Reply:
Ah. That one seems to have been done automatically by rails when I ordered it to remove the Privilege model.

Reply:
Ah, that'd do it. I tend to delete stuff manually rather than getting Rails to do it, for reasons just like this.